updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,171 @@
+
+// Mirroring from mirror to disk
+include::modules/oc-mirror-mirror-to-disk.adoc[leveloffset=+4]
+
+// Mirroring from disk to mirror in a disconnected environment
+include::modules/oc-mirror-disk-to-mirror.adoc[leveloffset=+4]
+
+// Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+include::modules/oc-mirror-updating-cluster-manifests.adoc[leveloffset=+2]
+
+[id="updating-mirror-registry-content"]
+=== Keeping your mirror registry content updated
+
+After your target mirror registry is populated with the initial image set, be sure to update it regularly so that it has the latest content. You can optionally set up a cron job, if possible, so that the mirror registry is updated on a regular basis.
+
+Ensure that you update your image set configuration to add or remove {product-title} and Operator releases as necessary. Any images that are removed are pruned from the mirror registry.
+
+// About updating your mirror registry content
+include::modules/oc-mirror-updating-registry-about.adoc[leveloffset=+3]
+
+// Updating your mirror registry content
+include::modules/oc-mirror-differential-updates.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-image-set-examples_mirroring-ocp-image-repository[Image set configuration examples]
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-image-set-partial[Mirroring an image set in a partially disconnected environment]
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-image-set-full[Mirroring an image set in a fully disconnected environment]
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-updating-cluster-manifests_mirroring-ocp-image-repository[Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster]
+
+// Performing a dry run
+include::modules/oc-mirror-dry-run.adoc[leveloffset=+2]
+
+// Mirroring Operator images in OCI format
+include::modules/oc-mirror-oci-format.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../operators/admin/olm-managing-custom-catalogs.adoc#olm-managing-custom-catalogs-fb[File-based catalogs]
+
+// Image set configuration parameters
+include::modules/oc-mirror-imageset-config-params.adoc[leveloffset=+2]
+
+// Image set configuration examples
+include::modules/oc-mirror-image-set-config-examples.adoc[leveloffset=+2]
+
+// Command reference for oc-mirror
+include::modules/oc-mirror-command-reference.adoc[leveloffset=+2]
+
+include::modules/update-mirror-repository.adoc[leveloffset=+1]
```

Comment:
I'd make sure that this section is really all you needed to do the oc adm release version. Compared with this section on it: https://55448--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-installation-images.html

Not sure if it's missing anything else from here or not

Reply:
Before I started documenting oc mirror as an alternative option, this one section was all the updating documentation had for the explicit mirroring process. 

But to your point, wouldn't hurt for me to double check whether any of the sections from your linked page should be included too! Will do

Reply:
Okay then that might be it. I was looking at using oc adm release for the installation process - so maybe it is different. Then you're probably right that this might be sufficient as is!